<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCashOp</class>
 <widget class="QDialog" name="DlgCashOp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Money in/out</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5LineEdit" name="leAmount"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Purpose</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEdit" name="lePurpose"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lbInOut">
     <property name="styleSheet">
      <string notr="true">color:red</string>
     </property>
     <property name="text">
      <string>IN/OUT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
